src/main/java/Lubot.java

```diff
@@ -0,0 +1,202 @@
+import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
nit: Should imports have lexicographically increasing order?

```suggestion
import java.util.ArrayList;
import java.util.Scanner;
```